### PR TITLE
Updated evaluation scores of the OcrPerturbation and the description of the TokenReplacement transformation

### PR DESCRIPTION
**Pull request by mnamysl**
In this PR, evaluation scores of the OcrPerturbation were updated and the description of the TokenReplacement transformation was corrected.

### CODE REVIEW

**Review by AbinayaM02 — COMMENTED** [review]
@mnamysl : Thank you for updating the evaluation scores. Could you please pull the latest code once? I can approve the PR once it's done.

**Comment by mnamysl** [comment]
@AbinayaM02 : Done. Thank you for your feedback.

**Review by AbinayaM02 — APPROVED** [review]
Approving the changes to evaluation scores of OcrPerturbation and README of TokenReplacement transformations.

**Pull request merged**
